client/state/directly/actions.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Internal dependencies
+ */
+import {
+	initialize as initializeDirectly,
+	askQuestion as askDirectlyQuestion,
+	maximize as maximizeDirectly,
+	minimize as minimizeDirectly,
+	openAskForm as openDirectlyAskForm,
+} from 'lib/directly';
+import {
+	DIRECTLY_ASK_QUESTION,
+	DIRECTLY_INITIALIZE,
+	DIRECTLY_MAXIMIZE,
+	DIRECTLY_MINIMIZE,
+	DIRECTLY_OPEN_ASK_FORM,
+} from 'state/action-types';
+
+export const askQuestion = ( questionText, name, email ) => ( dispatch ) => {
+	dispatch( { type: DIRECTLY_ASK_QUESTION, questionText, name, email } );
+	askDirectlyQuestion( questionText, name, email );
+};
```

Comment:
[These started out in middleware](https://github.com/Automattic/wp-calypso/pull/11306/files/080a430c6929239a5a80d0a3c3563c019e41dc84#diff-ee57c146294569bf59fcf197b7168dbc), but at the time `directly.initialize()` was an async function that took a callback and dispatched a new success or failure action. I was advised to move them to Action Creators to avoid dispatching new actions within middleware.

So just confirming — now that the Directly functions are fully synchronous and we're not responding to callbacks with more actions, is middleware the best place for these? And if we were to modify them to accept callbacks for success/failure, should they be moved back out of middleware into action creators?

/cc @omarjackman